CodeStarterCamp_Week3/main.swift

```diff
@@ -8,5 +8,50 @@
 
 import Foundation
 
-print("Hello, World!")
+struct Person {
+    let name: String
+    var age: Int
+    var money: Int
 
+    mutating func buyCoffee(myMemu: Coffee) {
```

Comment:
```suggestion
    mutating func buyCoffee(from menu: Coffee) {
```

여기는 `from`이라는 전달인자 레이블이 적절해보이네요!